apps/drec-ui/env.sh

```diff
@@ -33,6 +34,6 @@ for i in "${REQUIRED_VARIABLES[@]}"
     if [[ $i == $last ]]; then
         echo "$varname=$value " >> ./.env.local
       else
-        echo "$varname=$value," >> ./.env.local
+        echo "$varname=$value" >> ./.env.local
```

Comment:
hm?

Reply:
When this compiled it would add to .env.local in react a ',' after each variable.
Example: 
REACT_APP_BACKEND_URL=http://wsl:3040,